docs/api.yaml

```diff
@@ -0,0 +1,760 @@
+        '401':
+          description: Permission error
+        '404':
+          description: Invalid ID supplied
+  /data_accounting/write/store_signed_tx:
+    post:
+      tags:
+        - internal
+      summary: Store signed transaction
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                rev_id:
+                  type: integer
+                  example: 1
+                signature:
+                  type: string
+                  example: '0xabc...cba'
+                public_key:
+                  type: string
+                  example: '0xabc...cba'
+                wallet_address:
+                  type: string
+                  example: '0xabc...cba'
+      responses:
+        '200':
+          description: successful operation
+        '401':
+          description: Permission error
+        '500':
+          description: Internal server error
+  /data_accounting/write/store_witness_tx:
+    post:
+      tags:
+        - internal
+      summary: Store witness transaction
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                witness_event_id:
+                  type: integer
+                  example: 1
+                account_address:
+                    type: string
+                    example: '0xabc...cba'
+                transaction_hash:
+                    type: string
+                    example: 'abc...cba'
+                witness_network:
+                    type: string
+                    example: 'goerli'
+      responses:
+        '200':
+          description: successful operation
+        '401':
+          description: Permission error
+        '404':
+          description: Invalid input data
+        '500':
+            description: Internal server error
+  /data_accounting/transclusion/update_hash:
+    post:
+      tags:
+        - internal
+      summary: Update hash for transclusion
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                page_title:
+                  type: string
+                  example: 'Main_Page'
+                resource:
+                  type: string
+                  example: 'Template:A'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: true
+        '401':
+          description: Permission error
+        '404':
+          description: Invalid input data
+        '500':
+            description: Internal server error
+  /data_accounting/delete_revisions:
```

Comment:
This API needs to be exposed  externally to enable a feature of the Guardian to delete revisions based on their verification hash.

Reply:
This is for a seperate issue, which we track here https://github.com/inblockio/mediawiki-extensions-Aqua/issues/366